tests/non_search/setup_helpers.py

```diff
@@ -147,6 +149,39 @@ def setup_with_two_docs(db: Session):
     )
 
 
+def setup_with_documents_large_one_family(
+    documents_large: list[Dict],
+    db: Session,
+):
+    # Collection
+    collection1, collection2 = get_default_collections()
+    add_collections(db, collections=[collection1])
+
+    # Family + Document + events
+    family1, family2 = get_default_families()
+
+    family1["metadata"] = {
+        "topic": "Mitigation",
+        "sector": "Economy-wide",
+    }
+
+    family1["documents"] = documents_large[:20]
+    family2["documents"] = documents_large[20:]
+    add_families(db, families=[family1, family2])
+
+    # Collection - Family
+    link_collection_family(
+        db,
+        [
+            (collection1["import_id"], family1["import_id"]),
+        ],
+    )
+
+    # TODO Geographies
```

Comment:
Do these still need to go in?

Reply:
In my head we need to ensure that any schema / data change that might propagate through from these queries in this function is caught: 

`browse_rds_families` 

![image](https://github.com/user-attachments/assets/2d944337-ae2c-4cee-818a-abe8889d867f)

So we would need geos, corpus, org, family_corpus (especially geo as this is what caused the bug)?

Reply:
It may also be good to get a test with two families.

Reply:
I've removed the `TODO`s.

There are some `Geography` and `FamilyGeography`s created [1]. Honestly, it's been so long (~2 weeks) since the bug, I've partially forgotten what the issue was, even when looking at https://github.com/climatepolicyradar/navigator-backend/pull/322.

[1] I printed them out during the test, to check.
```
All FamilyGeography entries:
Family Import ID: CCLW.family.1001.0, Geography: South Asia
Family Import ID: CCLW.family.2002.0, Geography: Afghanistan
{"written_at": "2024-09-16T13:03:21.510Z", "written_ts": 1726491801510698000, "msg": "Running pipeline family query", "type": "log", "logger": "app.core.ingestion.pipeline", "thread": "MainThread", "level": "INFO", "module": "pipeline", "line_no": 28, "correlation_id": "-"}
{"written_at": "2024-09-16T13:03:23.625Z", "written_ts": 1726491803625245000, "msg": "Running pipeline document query", "type": "log", "logger": "app.core.ingestion.pipeline", "thread": "MainThread", "level": "INFO", "module": "pipeline", "line_no": 62, "correlation_id": "-"}

Geographies for each state row:
Import ID: CCLW.document.i00000008.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000204.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000192.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000222.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000006.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000298.n0000, Geographies: ['So
uth Asia']
Import ID: CCLW.document.i00000214.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000067.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000287.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000026.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000247.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000030.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000280.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000038.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000058.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000144.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000153.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000002.n0000, Geographies: ['South Asia']
Import ID: CCLW.document.i00000066.n0000, Geographies: ['AFG']
Import ID: CCLW.document.i00000040.n0000, Geographies: ['AFG']
Import ID: CCLW.document.i00000305.n0000, Geographies: ['AFG']
Import ID: CCLW.document.i00000018.n0000, Geographies: ['AFG']
Import ID: CCLW.document.i00000290.n0000, Geographies: ['AFG']
```

Reply:
Going to get this in as is, for the sake of moving forward for now!